가운데-글자-가져오기.js

```diff
@@ -0,0 +1,17 @@
+function solution(s) {
+    //문자열 길이 length
+    //짝수면 가운데 글짜 두개 , 홀수면 가운대 글자 1개
+    
+    if (s.length % 2 ==0) {  //길이가 짝수면 
+        answer = s[s.length / 2 - 1]+ s[s.length / 2]; //가운데 글자 두개 
+    //예) 문자열의 길이가 8이라면 4-1과 4를 출력하라는 뜻이다. 
+
+    }
+    else {
+        answer = s[s.length / 2 -0.5]; // 5라는 숫자가 있으면 나누면 2.5  
+    }
+    
+      return answer;
+    }
+    
+    
```

Comment:
잘 푸셨는데 전체적으로 리팩터링이 필요해보여요

if 또는 else 문의 실행문이 한 줄 일때는 중괄호를 생략해 줘도 돼요.
```suggestion
function solution(s) {
  let answer = "";
  if (s.length % 2 == 0) {
    answer = s[s.length / 2 - 1] + s[s.length / 2];
  } else answer = s[s.length / 2 - 0.5];
  
  return answer;
}

```




추가적으로 아래 방법으로 풀 수도 있으니 같이 공부해 보면 좋을 것 같습니다~


```javascript
function solution(s) {
    const sLen = s.length;
    const center = Math.floor(sLen / 2);
    
    return (sLen % 2) ? s.slice(center, center + 1) : s.slice(center - 1, center + 1);
}
```